iotics/lib/identity/api/high_level_api.py

```diff
@@ -85,16 +134,42 @@ def create_twin_with_control_delegation(self, twin_seed: bytes, twin_key_name: s
     def get_ownership_of_twin_from_registered_identity(self, twin_registered_identity: RegisteredIdentity,
                                                        new_owner_registered_identity: RegisteredIdentity,
                                                        new_owner_key_name: str):
+        """
+        Get Ownership of a twin.
```

Comment:
perhaps work saying like "get ownership of a twin by adding a key you control" or something